device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -86,6 +92,12 @@ class GroupsResource(namespaceExtractor: Directive1[Namespace])
         (get & path("devices")) {
           getDevicesInGroup(groupId)
         } ~
+        (post & pathPrefix("devices") & namespaceExtractor & extractUuid) { (ns, deviceId) =>
```

Comment:
Is it possible to add a device from another namespace? Or will magic database action prevent this?